src/video/windows/SDL_windowsrawinput.h

```diff
@@ -25,6 +25,8 @@
 
 extern bool WIN_SetRawMouseEnabled(SDL_VideoDevice *_this, bool enabled);
 extern bool WIN_SetRawKeyboardEnabled(SDL_VideoDevice *_this, bool enabled);
-extern bool WIN_RefreshRawInputEnabled(SDL_VideoDevice *_this);
+
+#define WIN_RAWKEYBOARD_FLAG_ENUM_NOHOTKEYS 0
+extern bool WIN_SetRawKeyboardFlag(SDL_VideoDevice *_this, Uint32 flag, bool enabled);
```

Comment:
The enum made more sense.

I think this should just be:

```suggestion
extern bool WIN_SetRawKeyboardNoHotkeys(SDL_VideoDevice *_this, bool enabled);
```

I think the simplification @slouken mentioned in https://github.com/libsdl-org/SDL/pull/13066#discussion_r2312489413 was removing the enum(same as my suggestion), not making it an integer.